### DISCUSSION TITLE
is comp_valid_licenses correct?

### DISCUSSION THREAD

**Discussion by yan09test01** [interlynk-io/sbomqs#309] [Q&A] [answered]
Thank you for making a tool for checking sbom quality.
I'm asking because "comp_valid_licenses" is strange while using it.
If I enter a value defined in the SPDX licenses in the license id, the score does not come out, and if I enter an undefined value, the score goes up. Can you check it?
Thank you.

**Comment by viveksahu26** [comment]
Yeah, you are right. I have reproduce this issue and raised a issue for the same.

**Reply by riteshnoronha** [reply, marked as answer]
fixed with v0.1.8
